nextjs/server-side-rendering/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "server-side-rendering-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@appwrite.io/pink": "^0.2.0",
+    "@appwrite.io/pink-icons": "^0.2.0",
+    "luke-node-appwrite-ssr": "^0.0.5",
```

Comment:
As you have switched from "luke-node-appwrite-ssr" to "node-appwrite" (https://github.com/appwrite/demos-for-react/pull/28/commits/a8d07e726317ffc338ecc9aeddfe03ff86d09326), you should ensure that the package.json is also changed.

Reply:
We need to wait for SDK public release for this 👍